packages/module/src/WarningModal/WarningModal.tsx

```diff
@@ -33,9 +39,24 @@ const WarningModal: React.FunctionComponent<WarningModalProps> = ({
   checkboxLabel='I understand that this action cannot be undone',
   confirmButtonVariant = ButtonVariant.primary,
   ouiaId = 'WarningModal',
+  confirmationInputProps,
+  confirmationInputLabel = 'Type to confirm',
+  confirmationText,
```

Comment:
```suggestion
  confirmationText,
  confirmationInputLabel = <>Type <strong>{confirmationText} </strong> to confirm the action:</>,
```
I would keep your previous default there - it makes sense to have the confirmation text present

Reply:
fixed